libs/ui/src/theme/theme.ts

```diff
@@ -1,167 +1,226 @@
-import {
-  amber,
-  amberDark,
-  grass,
-  grassDark,
-  indigo,
-  indigoDark,
-  sky,
-  skyDark,
-  slate,
-  slateDark,
-  tomato,
-  tomatoDark,
-  gray,
-  grayDark,
-} from '@radix-ui/colors';
 import { Theme } from '../types/theming';
+
+import {
+  primary,
+  primaryDark,
+  secondary,
+  secondaryDark,
+  neutral,
+  neutralDark,
+  utility,
+  utilityDark,
+} from './global/colors';
+
 import {
   PrimaryDarkButton,
   PrimaryLightButton,
   SecondaryDarkButton,
   SecondaryLightButton,
   TertiaryDarkButton,
   TertiaryLightButton,
-} from './component/button';
+} from './atoms/button';
+import { FieldTheme, FieldThemeDark } from './atoms/field';
 
 export const defaultDarkTheme: Theme = {
```

Comment:
I would be interested in the idea of creating an atomic design folder structure for the theme?
I'll put togeather a hack md file to cover the pros & cons.

Reply:
I like the rigour, but we're in ultra-budget mode now. Probably best to leave it for now.